### DISCUSSION TITLE
404 페이지 (v)

### DISCUSSION THREAD

**Discussion by wjsdncl** [Part3-Team15/taskify#152] [General]
잘못된 주소로 이동했을 때 404 페이지에서 리다이렉션 하는게 좋을까요 아니면 모달로 리다이렉션 처리하는게 좋을까요?

**Comment by un0211** [comment]
그냥  바로 404로 이동하면 되지 않을까요??

**Reply by wjsdncl** [reply]
저도 404 페이지가 있는게 좋을거 같아요 오늘 팀미팅에서 의견 들어보고 만들기로 정해지면 제가 만들겠습니다 🫡
